modules/installing-aws-load-balancer-operator-cli.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-installing-aws-load-balancer-operator-cli_{context}"]
+= Installing the AWS Load Balancer Operator with the command line
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* Your cluster is configured with AWS as the platform type and cloud provider.
+
+* You are logged into the command line client `oc`.
+
+.Procedure
+
+. Create the `aws-load-balancer-operator` namespace:
++
+[source,terminal]
+----
+oc apply -f namespace.yaml
+cat namespace.yaml
+----
++
+Edit the namespace to include:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: aws-load-balancer-operator
+----
+
+. Create the `credentialsrequest` file:
++
+[source, terminal]
+----
+oc apply -f credentialsrequest.yaml
```

Comment:
Same for all of these to only have a single command in each code block, and making sure to have a "$" prompt.